requirements.txt

```diff
@@ -3,3 +3,4 @@ sqlalchemy >= 1.4
 click
 pyyaml
 pyld
+pydantic >= 2, < 3
```

Comment:
I don't think this change should be part of this PR. The pydantic requirements should be added on #25 shouldn't they?

Reply:
There was some simultaneous development where I made the same change in both branches. I'll remove this and make the change in the Pydantic branch.